docs/scripts/generate_references.py

```diff
@@ -0,0 +1,59 @@
+import mkdocs_gen_files
+import os
+from pprint import pprint
+from griffe.dataclasses import Kind
+from griffe.dataclasses import Module
+from griffe.dataclasses import Alias
+
+
+pyhand = mkdocs_gen_files.config['plugins']['mkdocstrings'].get_handler('python')
+module_name = 'CosmoTech_Acceleration_Library'
+
+griffed_module = pyhand.collect(module_name, {})
+
+# pprint(griffed_module.as_dict())
+
+
+def yield_module_member(module: Module) -> list:
```

Comment:
what about files with classes AND functions ?

Reply:
Done, i've also change layout to display correctly the Table of content and fuse pages which are under the same file.